ci-operator/config/containers/libpod/master.yaml

```diff
@@ -17,13 +17,13 @@ build_root:
 tests:
 - artifact_dir: /tmp/artifacts
   as: validate
-  commands: ARTIFACT_DIR=/tmp/artifacts make validate
+  commands: ARTIFACT_DIR=/tmp/artifacts GOPATH=/go make validate
```

Comment:
The build image already sets GOPATH, if you aren’t seeing it that means someone regressed the image.  @stevekuznetsov I might suspect the build root changes.

Reply:
This is based on a Fedora image and not the openshift centos one.  Are you setting an image ENV for GOPATH? If so, I could do that on the Fedora image too.

Reply:
Wait, `clonerefs` upsteam will set up GOPATH to be correct -- if the clone for `ci-operator` doesn't own this, it needs to. It should be invisible to the user.

Reply:
I don't think it exports it though?

Reply:
Yeah that is a bug on our end :(